custom_components/tahoma/select.py

```diff
@@ -59,7 +59,7 @@ def __init__(
     @property
     def current_option(self):
         """Return the selected entity option to represent the entity state."""
-        return self.device.states.get(CORE_OPEN_CLOSED_PEDESTRIAN_STATE).value
+        return self.device.states[CORE_OPEN_CLOSED_PEDESTRIAN_STATE].value
```

Comment:
I also changed this, since it will throw an error anyways if it returns None. Thus it doesn't make sense to use `.get()` instead of calling it directly.

Reply:
I don't think it works it's not an array. It's why we use the get method.

Reply:
You could be right, I will revert this, since I cannot test it.